### PR TITLE
Team2 project management with GitHub

### PR DESCRIPTION
**Pull request by hinckleyc**
Added information to:
 Assigning team members to issues in the GitHub issue tracker,
 Merging a branch or a fork to another branch or fork,
 Creating and reviewing a pull request,
 Conduct.
Please let me know if there is anything that needs to be fixed.

### CODE REVIEW

**Review by enpuyou — CHANGES_REQUESTED** [review]
Please conform to the style guide here: https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown

File: assessment.md [hunk 1]
```diff
@@ -36,11 +36,34 @@ Project Management is Covered by Team 2
...
 * Creating and using a repository fork
 * Merging a branch or a fork to another branch or fork
+  * N = Not making a branch when it your job
+  * I = when merging a branch or a fork to another branch or fork does not work
+  * A = when merging a branch or a fork to another branch or fork works only
+  sometimes
+  * G = when merging a branch or a fork to another branch or fork works most of
+  the time
+  * E = when merging a branch or a fork to another branch or fork works all the
+  time
```
Comment (corlettim) [inline assessment.md]: I recommend rewording these. I also suggest making a standard format with capitalizing after the equal sign or not

**Comment by koscinskic** [comment]
I have resolved the requested changes concerning emoji style of assessment.

File: assessment.md (hunk 1 above)
Comment (hodevin) [reply assessment.md]: It should be capitalized in this case.

**Comment by noorbuchi** [comment]
Hi, I'm a member of the markdown team and I've been trying to review some of the pull requests and make sure that the branches adhere to the markdown style in the markdown repository . There is a small issue with your pull request in which you are using bold in your writing. Please let me know if you any questions about markdown usage. Thanks!

File: assessment.md [hunk 2]
```diff
@@ -32,15 +32,124 @@
...
+  actively engages in and/or initiates a team review and discussion.
+
 * Using the GitHub issue tracker
+  * 😞 = do not use the issue tracker
+  * 😕 = occasionally use the issue tracker, issue descriptions are confusing
+  * 😐 = use the issue tracker, but will still occasionally use slack or
+  other communication, issue descriptions occasionally confusing
+  * 😃 = use the issue tracker, issue descriptions not as in depth as they
+  should be
+  * 😎 = use the issue tracker, issue descriptions are in depth and clear
```
Comment (enpuyou) [inline assessment.md]: These two sections do not conform to the style guide. The first letter after equal sign should also be uppercase.

File: assessment.md [hunk 3]
```diff
@@ -32,15 +32,124 @@
...
+  * :neutral_face: = Team member was assigned to issue and team member tried to
+    assist but no change was made.
+  * :smiley: = Team member was assigned to issue and issue was resolved.
+  * :sunglasses: = Team member was assigned to issue and issue was resolved and
+    understood.
+
 * Creating and assigning labels to issues in the GitHub issue tracker
+  * :disappointed: = Doing nothing.
+  * :confused:= Assigning but not clarifying tasks.
+  * :neutral_face: = Assigned tasks that are clear but are useless.
```
Comment (enpuyou) [inline assessment.md]: Why are both "assigning" and "assigned" both used here?

File: assessment.md [hunk 4]
```diff
@@ -32,15 +32,124 @@
...
+  * :disappointed: = No team member was assigned to issue.
+  * :confused: = Team member was assigned to issue but no response was given.
+  * :neutral_face: = Team member was assigned to issue and team member tried to
+    assist but no change was made.
+  * :smiley: = Team member was assigned to issue and issue was resolved.
+  * :sunglasses: = Team member was assigned to issue and issue was resolved and
+    understood.
+
 * Creating and assigning labels to issues in the GitHub issue tracker
+  * :disappointed: = Doing nothing.
```
Comment (enpuyou) [inline assessment.md]: I think it should be something more descriptive than just saying "doing nothing"

File: assessment.md [hunk 5]
```diff
@@ -32,15 +32,124 @@
...
+    group members who need assistance, ensuring that use of branches is
+    consistent across team members.
+
+A software engineer demonstrates mastery of this technical skill at the
+:sunglasses: level by communicating with their team to determine a naming scheme
+for branches, and the plan is executed as agreed upon. The branch(es) being used
+should be named according to their role (i.e.
+team2-project-management-with-github), and once that role is complete/no longer
+necessary, a pull request should be created to merge the branch with the main
+branch.
```
Comment (enpuyou) [inline assessment.md]: This section should be removed according to the markdown assessment template

File: assessment.md [hunk 6]
```diff
@@ -32,15 +32,124 @@
...
 * Creating and using a repository fork
+  * 😞 = Didn't create or utilize repository forks.
+  * 😕 = Created unclear repository forks.
+  * 😐 = Created poorly named repository forks.
+  * 😃 = Created useful named repository forks
+  * 😎 = Created useful and descriptive repository forks
+
 * Merging a branch or a fork to another branch or fork
+
+  * :disappointed: = Not making a branch when it your job
```
Comment (enpuyou) [inline assessment.md]: This sentence needs to be fixed

File: assessment.md [hunk 7]
```diff
@@ -32,15 +32,124 @@
...
+
+  * :disappointed: = Not creating a pull request
+  * :confused: = Does not mention the issues it is trying to close with a subpar
+    description
+  * :neutral_face: = Mentions related issues it is trying to close and describes
+    the changes
+  * :smiley: = Mentions most related issues it is trying to close, describes
+    most changes and the purpose of the pull request
+  * :sunglasses: = Mentions all related issues it is trying to close, describes
+    all changes made and passes Travis CI
```
Comment (enpuyou) [inline assessment.md]: This section did not address "reviewing a pull request"

File: assessment.md [hunk 8]
```diff
@@ -32,15 +32,124 @@
...
+    as well as a brief description or list of changes if a commit contains a
+    large volume of changes. Additionally, guidance is provided to the group
+    ensuring members are following a consistent and uniform commit structure.
+
+A software engineer demonstrates the mastery of this technical skill at the
+:sunglasses: level by engaging with the team to determine an appropriate commit
+scheme and structure (e.g. action-content-file or equally descriptive format),
+consistently adhering to said structure, and adding commit descriptions if
+deemed necessary (e.g. Adding a list of feature additions / changes / removals
+or rationale for said actions).
```
Comment (enpuyou) [inline assessment.md]: This section should be removed

**Review by enpuyou — CHANGES_REQUESTED** [review]
This PR is not up to standard and many places below have to be fixed before merging. In general, there are inconsistent uses of the period at the end of each bullet point which should all be removed. All sections should also use present tense and third-person narrative

**Comment by enpuyou** [comment]
Most commit messages in your group are also not up to standard in terms of the use of tense and uppercase, while the content was somewhat descriptive. Please review the standard for commit messages and write better commit messages in the future

File: assessment.md (hunk 5 above)
Comment (koscinskic) [reply assessment.md]: This specific issue (The presence of this section) is currently being addressed as an issue in the Markdown repository.

File: assessment.md (hunk 8 above)
Comment (koscinskic) [reply assessment.md]: This specific issue (The presence of this section) is currently being addressed as an issue in the Markdown repository.

Comment (enpuyou) [reply assessment.md]: That's fine. I would suggest you fix the other issues first, and once the issue your team raised is being addressed, we can continue from there to work and merge this PR

**Comment by gkapfham** [comment]
Hello @hinckleyc, this PR is not up-to-date with the master branch. Can you please do the update and then confirm that everything is still working correctly? Make sure that you finish this task today, okay?

File: assessment.md [hunk 9]
```diff
@@ -136,15 +136,113 @@
...
+    group members who need assistance, ensuring that use of branches is
+    consistent across team members
+
 * Creating and using a repository fork
+
+  * :disappointed: = Didn't create or utilize repository forks.
+  * :confused: = Created unclear repository forks.
+  * :neutral_face: = Created poorly named repository forks.
+  * :smiley: = Created useful named repository forks
+  * :sunglasses: = Created useful and descriptive repository forks
```
Comment (enpuyou) [inline assessment.md]: This part is still in the past tense

**Review by enpuyou — CHANGES_REQUESTED** [review]
One more fix

**Review by enpuyou — APPROVED** [review]

**Review by stephensonc — APPROVED** [review]

**Review by koscinskic — APPROVED** [review]

**Pull request merged**